firmware/quadruna/VC/src/app/app_powerManager.c

```diff
@@ -1,52 +1,211 @@
 #include "io_efuse.h"
 #include "app_powerManager.h"
-#include "app_canRx.h"
+#include "app_canTx.h"
+
+const int   DEBOUNCE_TIMER_LIMIT = 100;
+const float TIMER_ATTEMPTS_LIMIT = 100.0;
 
 typedef struct
 {
-    bool efuses[NUM_EFUSE_CHANNELS];
+    RetryConfig retry_configs[NUM_EFUSE_CHANNELS];
 } PowerStateConfig;
 
+static RetryData retry_data[NUM_EFUSE_CHANNELS];
+
 static const PowerStateConfig power_states_config[NUM_POWER_STATES] = {
-    [POWER_MANAGER_SHUTDOWN] = {
-        .efuses = {
-            [EFUSE_CHANNEL_SHDN] = true,
-            [EFUSE_CHANNEL_LV] = true,
-            [EFUSE_CHANNEL_PUMP] = false,
-            [EFUSE_CHANNEL_AUX] = false,
-            [EFUSE_CHANNEL_INV_R] = false,
-            [EFUSE_CHANNEL_INV_L] = false,
-            [EFUSE_CHANNEL_TELEM] = true,
-            [EFUSE_CHANNEL_BUZZER] = false,
-        }
-    },
-    [POWER_MANAGER_DRIVE] = {
-        .efuses = {
-            [EFUSE_CHANNEL_SHDN] = false,
-            [EFUSE_CHANNEL_LV] = true,
-            [EFUSE_CHANNEL_PUMP] = true,
-            [EFUSE_CHANNEL_AUX] = true,
-            [EFUSE_CHANNEL_INV_R] = true,
-            [EFUSE_CHANNEL_INV_L] = true,
-            [EFUSE_CHANNEL_TELEM] = true,
-            [EFUSE_CHANNEL_BUZZER] = true,
-        }
-    }
+    [POWER_MANAGER_SHUTDOWN] = { .retry_configs = { [EFUSE_CHANNEL_SHDN]   = { .efuse_state    = true,
```

Comment:
Can you set the `min_needed_current` all to be zero for now? I think this'll be more convenient until the VC is in the car and we can test this properly. Otherwise it's gonna be constantly faulting while on the bench.

Reply:
I have it at 0.5 for unit testing, so I'll change it to 0 and comment out the tests.